fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -83,6 +83,10 @@ class PythonUtils {
 	    	{
 	    		return "[\"" + str.substring(1, str.length()-1).replaceAll(", ",",").replaceAll(",","\", \"") + "\"]";
 	    	}
+	    	else if (str.contains(" "))
+	    	{
+	    		return "[\"" + str.substring(1, str.length()-1).replaceAll(" "," ")+"\"]";
```

Comment:
Does `replaceAll(" "," ")` do anything useful?  Replacing spaces with spaces seems unnecessary.

Reply:
Agree. No need to replace space. done the change.